pkg/ui/src/components/NoteButton/NoteButton.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+
+import ArrowCircleLeft from "@assets/ArrowCircleLeft.svg";
+import ArrowCircleRight from "@assets/ArrowCircleRight.svg";
+
+// #region types
+export interface NoteButtonProps
+  extends React.HTMLAttributes<HTMLButtonElement> {
+  type?: NoteButtonType;
+  label?: NoteButtonLabel | Warehouse;
+  onClick?: () => void;
+}
+
+export enum NoteButtonLabel {
+  RemoveBooks = "Remove Books",
+}
+
+export enum NoteButtonType {
+  In = "in",
+  Out = "out",
+}
+
+export enum Warehouse {
+  Scolastica = "scolastica",
+  Nuovo = "nuovo",
+  Varia = "varia",
+}
+// #endregion types
+
+// #region color
+const colorClassesLookup = {
+  [NoteButtonType.In]: ["text-white", "bg-green-400", "hover:bg-green-500"],
+  [NoteButtonType.Out]: ["text-white", "bg-red-400", "hover:bg-red-500"],
+};
+
+// Classes for focus pseudo-class are the same regardless of color prop
+const focusClasses = [
+  "focus:ring-2",
+  "focus:ring-indigo-500",
+  "focus:ring-offset-2",
+  "focus:ring-offset-white",
+];
```

Comment:
This seems unnecessary as it's not specified in the spec (nor used in this context I reckon) and (I'm not a designer, but) I think this color, indigo, wouldn't really work well with the other colors)